src/lib.rs

```diff
@@ -393,6 +393,17 @@ pub enum _CastType {
 }
 
 impl DstLayout {
+    /// The maximum theoretic possible alignment of a type (1).
+    ///
+    /// (1) Per https://doc.rust-lang.org/reference/type-layout.html#the-alignment-modifiers:
```

Comment:
Assuming that your use of `(1)` is to avoid `[1]` being formatted as Markdown:

```suggestion
    /// The maximum theoretic possible alignment of a type \[1\].
    ///
    /// \[1\] Per https://doc.rust-lang.org/reference/type-layout.html#the-alignment-modifiers:
```

Alternatively, since we never actually rustdoc this, you could just change this to be a non-doc comment (and also you could change `2<sup>29</sup>` to `2^29`, which is more readable in the source code).